src/index.js

```diff
@@ -1,9 +1,15 @@
-const { initialize, loggers } = require('@asymmetrik/node-fhir-server-core');
+const { loggers } = require('@asymmetrik/node-fhir-server-core');
 const mongoUtil = require('./util/mongo');
 const { buildConfig } = require('./util/config');
+const { initialize } = require('./server/server');
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+app.use(bodyParser.json({ limit: '50mb', type: 'application/json+fhir' }));
```

Comment:
So it turns out `body-parser' is actually included in express now and can be used directly. This should solve the deprecation warning. Also `body-parser` doesn't need to be included in the package.json.
```suggestion
const express = require('express');

const app = express();
app.use(express.json({ limit: '50mb', type: 'application/json+fhir' }));
```